<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dance</class>
 <widget class="QMainWindow" name="dance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="3,1">
    <item row="0" column="0">
     <widget class="QLabel" name="labelgif">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Gif</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_ruch">
        <property name="text">
         <string>Ruch</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_x">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_y">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_z">
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_danex">
            <property name="text">
             <string>DaneX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_daney">
            <property name="text">
             <string>DaneY</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_danez">
            <property name="text">
             <string>DaneZ</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Volume</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="Slider_Volume">
          <property name="value">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelpkt">
        <property name="text">
         <string>Punkty</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1203</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <widget class="QMenu" name="menu_start">
     <property name="title">
      <string>Start</string>
     </property>
     <addaction name="action_Freestyle"/>
     <addaction name="action_Standard"/>
    </widget>
    <addaction name="menu_start"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Freestyle">
   <property name="text">
    <string>Freestyle</string>
   </property>
  </action>
  <action name="action_Standard">
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
